src/src/attendances/attendances.core.service.ts

```diff
@@ -241,10 +241,17 @@ export class AttendancesCoreService {
 				  }
 			  }`;
 		try {
-			const data_list = (
-				await this.hasuraServiceFromServices.getData({ query })
-			)?.data?.attendance;
-			return data_list;
+			const result_response =
+				await this.hasuraServiceFromServices.getData({ query });
+			console.log('result_response', result_response);
```

Comment:
remove console logs pls

Reply:
console log on dev is required for testing and track cron job flow. after testing complete on dev server then i will remove all logs and sent PR again